test/AutoDiff/method.swift

```diff
@@ -98,45 +103,51 @@ MethodTests.test("instance method with generated adjoint, wrt only non-self") {
 
 // FIXME: Add a binary differential operator.
 //
-// MethodTests.test("instance method with generated adjoint, wrt self and non-self") {
-//   let g = #gradient({ (p: Parameter, o: Float) in p.multiplied(with: o) })
+// MethodTests.testWithLeakChecking(
+//   "instance method with generated adjoint, wrt self and non-self"
+// ) {
+//   let g = #gradient({ (p: Parameter, o: Tracked<Float>) in p.multiplied(with: o) })
```

Comment:
Just a comment: some tests are outdated:
- Reference to outdated `#gradient` expression.
- Reference to "adjoint" functions, which are now called "pullback "functions.
- Deprecated comment; multiple `@differentiable` attributes are now possible:
```
  // There is currently no way to define multiple custom VJPs wrt different
  // parameters on the same func, so we define a copy of this func per adjoint.
```

Filed [TF-929](https://bugs.swift.org/browse/TF-929) to track updating outdated `AutoDiff` tests.

Reply:
Ah, I generally did not make changes in the comments, but this should have slipped through.